test/max-requests.js

```diff
@@ -0,0 +1,118 @@
+const del = require('del');
+const mongo = require('emulate-mongo-2-driver');
+const { expect } = require('chai');
+const rp = require('request-promise');
+const enableDestroy = require('server-destroy');
+const apostropheMultisite = require('./app.js');
+const Promise = require('bluebird');
+
+describe('Apostrophe-multisite', function() {
+  describe('#dashboard', function() {
```

Comment:
minor jawn, but maybe this needs to be renamed

Reply:
'tis true about the jawn.

Reply:
Renamed.